### DISCUSSION TITLE
Formation of a tribunal of Filecoin-Plus stakeholders for dispute resolution.

### DISCUSSION THREAD

**Discussion by NSC-FIL** [filecoin-project/lotus#5781] [Ideas] [answered]
Clear guidelines to the rules regarding Filecoin-Plus should been established for example whether self-dealing should be allowed (see https://github.com/filecoin-project/notary-governance/issues/98) and other issues that affect the health of the Filecoin ecosystem. 

Once Filecoin-Plus rules have been agreed upon and published (preferably in a summary format), the proposal is to form a rotating or elected tribunal (i.e. a Notary, a rep from Protocol Labs, a Miner or other Filecoin community member(s)) to determine the validity of any complaints about infractions of these rules by another ecosystem participant.  The tribunal would act as judge in these disputes and determine whether there is an infraction and whether there should be a penalty.  Who is responsible and how to enforce the penalty should be discussed here as well.

**Comment by dkkapur** [comment, marked as answer]
@NSC-FIL - would recommend you file this here for additional discussion! https://github.com/filecoin-project/notary-governance/issues

**Reply by NSC-FIL** [reply]
https://github.com/filecoin-project/notary-governance/issues/105.
